js/chat_app/package.json

```diff
@@ -56,4 +56,4 @@
     "nodemon": "^2.0.20",
```

Comment:
Please, delete `package-lock.json`, and revert `yarn.lock` if the change wasn't intentional.